spec/features/admin/merchants/show_spec.rb

```diff
@@ -14,12 +14,12 @@
 
     expect(page).to have_link("Update Information")
     click_on "Update Information"
-
-    expect(current_path).to eq("/admin/merchants/#{merchant.id}/edit")
-    expect(page).to have_field(:name, with: "Willy's Bakery")
     
-    fill_in 'name', with: "Jimmy's Bakery"
-    click_on 'Submit'
+    expect(current_path).to eq("/admin/merchants/#{merchant.id}/edit")
+    expect(page).to have_field(:merchant_name, with: "Willy's Bakery")
```

Comment:
slick test of the form_with :model autofill

Reply:
Thanks for finding that syntax for the test Josh, super cool!